### PR TITLE
Add new endpoint function edit_profile

### PR DESCRIPTION
**Pull request by er-anup**
Add a new endpoint function edit_profile which takes parameter id and hash and allows the users with the capability of editing profile to edit their profile

### CODE REVIEW

File: cleansweep/plugins/volunteers/views.py [hunk 1]
```diff
@@ -191,3 +191,27 @@ def profile(id, hash):
...
+@plugin.route("/people/edit/<id>-<hash>/", methods=["GET", "POST"])
+@require_permission("volunteers.edit")
+def edit_profile(id, hash):
+    m = Member.find(id=id)
+    if not m or m.get_hash() != hash:
+        abort(404)
+
+    if request.method == "POST":
+        action = request.form.get('action')
+        if action == 'update':
```
Comment (RaghavPro) [inline cleansweep/plugins/volunteers/views.py]: Is there going to be any other action?

Comment (er-anup) [inline cleansweep/plugins/volunteers/views.py]: No , did you raise this because of GET parameter i will remove it.

Comment (RaghavPro) [inline cleansweep/plugins/volunteers/views.py]: Yeah, it seems pointless.

**Comment by anandology** [comment]
merged.

**Pull request closed without merging**